packages/components/src/color-picker/input-with-slider.tsx

```diff
@@ -31,6 +31,7 @@ export const InputWithSlider = ( {
 	return (
 		<HStack spacing={ 4 }>
 			<NumberControlWrapper
+				__next40pxDefaultSize
```

Comment:
This one also has an `size="__unstable-large"`.

Reply:
This is now addressed, removed `size="__unstable-large"` and used the new way to add the size.